sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_azureappconfigurationproviderbase.py

```diff
@@ -471,6 +472,8 @@ def _process_key_value_base(self, config: ConfigurationSetting) -> Union[str, Di
                     self._tracing_context.uses_ai_configuration = True
                 if APP_CONFIG_AICC_MIME_PROFILE in config.content_type:
                     self._tracing_context.uses_aicc_configuration = True
+                if SNAPSHOT_REF_CONTENT_TYPE in config.content_type:
+                    self._tracing_context.uses_snapshot_reference = True
```

Comment:
This is already tracked in `_process_configurations`, do we still need to check content type here?